src/api/auction.ts

```
@@ -141,3 +142,77 @@ export const deleteAuction = async (auctionId: string): Promise<void> => {
+ */
+export const uploadProductImages = async (productId: number, files: File[]): Promise<{
+  imageUrl: string;
+  originalFileName: string;
+  fileSize: number;
+  mimeType: string;
+}[]> => {
+  try {
+    const formData = new FormData();
+    
+    // 여러 파일을 files 파라미터로 추가
+    files.forEach((file, index) => {
```

ESLint

[error] 'index' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)